benchmark/imfilter.jl

```diff
@@ -0,0 +1,39 @@
+SUITE["imfilter"] = BenchmarkGroup(["imfilter"])
+SUITE["imfilter_naive"] = BenchmarkGroup(["imfilter", "naive implementation"])
+SUITE["imfilter_naive"]["CPU1"] = BenchmarkGroup(["CPU", "single thread"])
+SUITE["imfilter"]["CPU1"] = BenchmarkGroup(["CPU", "single thread"])
+# SUITE["imfilter"]["CPUThread"] = BenchmarkGroup(["CPU", "multi threads"])
+# SUITE["imfilter"]["CUDA"] = BenchmarkGroup(["GPU", "CUDA"])
+# SUITE["imfilter"]["ArrayFire"] = BenchmarkGroup(["GPU", "ArrayFire"])
+# SUITE["imfilter"]["OpenCL"] = BenchmarkGroup(["GPU", "OpenCL"])
+
+img_gray_256 = testimage("lena_gray_256")
+img_rgb_256 = testimage("lena_color_256")
+img_list_gray = [imresize(img_gray_256, ratio=ratio) for ratio in (0.5, 1.0, 2.0, 4.0)]
+img_list_rgb = [imresize(img_rgb_256, ratio=ratio) for ratio in (0.5, 1.0, 2.0, 4.0)]
+
+default_kernel = centered(ones((31, 31))./prod((31, 31)))
```

Comment:
What led you to choose the sizes here? (Both for the kernel and the images in the lists.) If running time is a concern we could do everything with smaller images and/or kernels.